tests/kernel/iop/libc/strtok.c

```diff
@@ -0,0 +1,39 @@
+#include <common-iop.h>
+#include <string.h>
+
+const char *getStringRep(const char *str) {
+	return str ? str : "null";
+}
+
+void doStrtok(char *s, const char *delim)
+{
+	printf("strtok('%s', '%s') -> ", getStringRep(s), getStringRep(delim));
```

Comment:
nit: The quotes are probably misleading, although I see why.  Should be fine.

-[Unknown]